src/FeedItem.php

```diff
@@ -147,6 +147,19 @@ public function validate()
         }
     }
 
+    public function hasEnclosure()
+    {
+        if (
+            isset($this->enclosure)
```

Comment:
@freekmurze I would like to know how you would write this `if` regarding your guidelines?
* Keep it as is
* return compound `isset() && ...` directly
* Split into three `if isset()`

In this simple case I think that all options are similar/equal readable. 🤔
But would like to hear your opinion.